cmake/morpheus_utils/package_config/ucx/Configure_ucx.cmake

```diff
@@ -20,168 +20,9 @@ include_guard(GLOBAL)
-    # UCT Library
-    add_library(ucx::uct STATIC IMPORTED GLOBAL)
-    set_target_properties(ucx::uct PROPERTIES
-      INTERFACE_INCLUDE_DIRECTORIES "$<BUILD_INTERFACE:${ucx_INSTALL_DIR}/include>;$<INSTALL_INTERFACE:include>"
-      INTERFACE_POSITION_INDEPENDENT_CODE "ON"
-    )
-    set_property(TARGET ucx::uct APPEND PROPERTY IMPORTED_CONFIGURATIONS RELEASE)
-    set_target_properties(ucx::uct PROPERTIES
-      IMPORTED_LOCATION_RELEASE "${ucx_INSTALL_DIR}/lib/libuct.a"
-      IMPORTED_SONAME_RELEASE "libuct.a"
-    )
-    add_dependencies(ucx::uct ucx)
-
-    # UCP Library
-    add_library(ucx::ucp STATIC IMPORTED GLOBAL)
-    set_target_properties(ucx::ucp PROPERTIES
-      INTERFACE_INCLUDE_DIRECTORIES "$<BUILD_INTERFACE:${ucx_INSTALL_DIR}/include>;$<INSTALL_INTERFACE:include>"
-      INTERFACE_POSITION_INDEPENDENT_CODE "ON"
-    )
-    set_property(TARGET ucx::ucp APPEND PROPERTY IMPORTED_CONFIGURATIONS RELEASE)
-    set_target_properties(ucx::ucp PROPERTIES
-      IMPORTED_LOCATION_RELEASE "${ucx_INSTALL_DIR}/lib/libucp.a"
-      IMPORTED_SONAME_RELEASE "libucp.a"
-    )
-    add_dependencies(ucx::ucp ucx)
-
-    # UCS Library
-    add_library(ucx::ucs STATIC IMPORTED GLOBAL)
-    set_target_properties(ucx::ucs PROPERTIES
-      INTERFACE_INCLUDE_DIRECTORIES "$<BUILD_INTERFACE:${ucx_INSTALL_DIR}/include>;$<INSTALL_INTERFACE:include>"
-      INTERFACE_POSITION_INDEPENDENT_CODE "ON"
-    )
-    set_property(TARGET ucx::ucs APPEND PROPERTY IMPORTED_CONFIGURATIONS RELEASE)
-    set_target_properties(ucx::ucs PROPERTIES
-      IMPORTED_LOCATION_RELEASE "${ucx_INSTALL_DIR}/lib/libucs.a"
-      IMPORTED_SONAME_RELEASE "libucs.a"
-    )
-    add_dependencies(ucx::ucs ucx)
-
-    # UCM Library
-    add_library(ucx::ucm STATIC IMPORTED GLOBAL)
-    set_target_properties(ucx::ucm PROPERTIES
-      INTERFACE_INCLUDE_DIRECTORIES "$<BUILD_INTERFACE:${ucx_INSTALL_DIR}/include>;$<INSTALL_INTERFACE:include>"
-      INTERFACE_POSITION_INDEPENDENT_CODE "ON"
-    )
-    set_property(TARGET ucx::ucm APPEND PROPERTY IMPORTED_CONFIGURATIONS RELEASE)
-    set_target_properties(ucx::ucm PROPERTIES
-      IMPORTED_LOCATION_RELEASE "${ucx_INSTALL_DIR}/lib/libucm.a"
-      IMPORTED_SONAME_RELEASE "libucm.a"
-    )
-    add_dependencies(ucx::ucm ucx)
-
-    # Combined ucx::ucx target
-    add_library(ucx::ucx INTERFACE IMPORTED)
-    set_target_properties(ucx::ucx PROPERTIES
-      INTERFACE_LINK_LIBRARIES "ucx::uct;ucx::ucp;ucx::ucs;ucx::ucm"
-    )
-
-    # Finally, add this to the style check dependencies
-    add_dependencies(${PROJECT_NAME}_style_checks ucx-install)
-  else()
-    # Found installed UCX. Make sure to call rapids_export_package without a version.
-    # Otherwise CMake fails with trying to add the dependency twice
-    rapids_export_package(
-      INSTALL ucx ${PROJECT_NAME}-core-exports
-      GLOBAL_TARGETS ucx ucx::ucp ucx::uct ucx_ucx ucx::ucp ucx::uct ucx::ucx
-    )
-
-  endif()
+  find_package(ucx ${UCX_VERSION} REQUIRED)
```

Comment:
Just to confirm, does this pull and compile correctly when ucx isn't in the environment?